lib/basicAuthenticationHandler/index.test.js

```diff
@@ -0,0 +1,69 @@
+import test from 'ava'
+import sinon from 'sinon'
+import { Miniflare } from 'miniflare'
```

Comment:
I had to delete this line for these tests to run as it's not found in package.json. It's also unused in this file.